deployment/network-operator/templates/operator.yaml

```diff
@@ -25,7 +25,6 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      control-plane: {{ .Release.Name }}-controller
       {{- include "network-operator.selectorLabels" . | nindent 6 }}
```

Comment:
so any change to this line will also break upgrade ?

Reply:
It looks like so